homework/tests/homework.spec.js

```diff
@@ -0,0 +1,230 @@
+// @ts-check
```

Comment:
Please update test file name to reflect what is being tested. E.g. `CalculatorPageTests.spec.js`